### ISSUE TITLE
Display user's workspaces at the top of the dashboard, separate from other workspaces.

### ISSUE THREAD

**Issue by karelklima** [opendata-mvcr/mission-control#132]
Users should see their workspaces right away when they open the application, other workspaces are not so important.

For starters we should display separately the workspaces that the user owns, but we could potentially create a "favorites" feature or similar so that the user may modify the priority list.

**Comment by psiotwo** [comment]
Good idea. Given the amount of parties coming into Assembly line in the next weeks we should consider some authorization model for workspaces.

**Comment by karelklima** [comment]
Good point. What about this?
- all the workspaces are read-only by default - e.g. users may access workspace details, they can see what R/W vocabularies are in that workspace and ideally they can open also TermIt and ontoGrapher in read-only mode
- instead of one workspace owner, we'd have a list of collaborators - e.g. the first "owner" would invite someone else and everyone would have the same rights; alternatively we could restrict destroy actions (like deleting workspace) to owners only, but that may be an overkill.
